helm/platform-health/templates/config_map.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: platform-health
```

Comment:
If all other resource names can be overridden, then this probably should be too.

Reply:
Moved to use `{{ .Release.Name }}` instead.